extensions/csrc/cuda/activation_kernel.cu

```diff
@@ -39,6 +39,9 @@ torch::Tensor silu_and_mul(const torch::Tensor& ins)
     auto ins_shape = ins.sizes().vec();
 
     ins_shape[0] = ins_shape[0]/2;
+    if (ins_shape[0] == 1) {
+      ins_shape.erase(ins_shape.begin());
+    }
```

Comment:
Wonder why adding modification here

Reply:
To avoid the view operation after silu and mul operation.

Reply:
might want to add a comment